addons/interact_menu/functions/fnc_renderIcon.sqf

```diff
@@ -38,7 +38,10 @@ if (_iconFile isEqualTo "") then {
     _iconFile = DEFAULT_ICON;
 };
 
-_text = if (GVAR(UseListMenu)) then {
+_text = if (
```

Comment:
First statement in the same line as opening bracket

Reply:
then indent would be bad

Reply:
leaving it as is, as discussed in slack